static_src/components/users.jsx

```diff
@@ -186,6 +199,33 @@ export default class Users extends React.Component {
     );
   }
 
+  get userParentEntityUserSelector() {
+    if (!this.isSpace) {
+      return null;
+    }
+
+    if (!this.currentUserIsSpaceManager) {
+      return (
+        <PanelDocumentation>
+          Org Managers and Space Managers can add current organization users into this space.
+        </PanelDocumentation>
+      );
+    }
+
+    return (
+      <OrgUsersSelector
```

Comment:
this component has a lot of attributes. for a lot of attributes, you can make this generic like `UserSelector`.
For OrgUserSelector, you probably wouldn't need all this. I'm leaning to keeping all of these and just rename to `UserSelector` and we can refactor for more generic cases later on.